### PR TITLE
Wrong behavior in FO when there is only one currency after deleted an other one

### LINKED ISSUE #28920: Wrong behavior in FO when there is only one currency
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add screenshots

After a fresh install we have only one currency => the drop-down list to change currency is not displayed in FO (Expected behaviour)
When we add a second currency => the drop-down list to change currency is displayed in FO header (Expected behaviour)
When we delete the second currency => The drop-down list is still visible (Bug) 

https://user-images.githubusercontent.com/31663369/176673343-e4c56623-b5ce-4b59-85eb-2ffecda91052.mp4

### Expected behavior

In case there is only one currency , the drop-down list to change currency will not appear in the FO header,

### Steps to reproduce

**In a fresh install PrestaShop:**

1. Go to FO and check that the currencies list is not visible in the header
2. Create a new currency
3. Go to FO and check that the drop-down list to change the currency is well displayed
4. Delete the created currency
5. Go to FO and check that the currencies list is still displayed in the header (it should be removed when we have only one currency) 
image

### PrestaShop version(s) where the bug happened

1.7.7.8, 1.7.8.6, 8.0.x

### PHP version(s) where the bug happened

7.4.28

### If your bug is related to a module, specify its name and its version

_No response_

### PR DESCRIPTION
**Pull request by ghost**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Wrong behavior in FO when there is only one currency after deleted an other one
| Type?             | bug fix
| Category?         |CO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes #28920.
| Related PRs       | 
| How to test?      | See issue
| Possible impacts? |

### CODE REVIEW

File: classes/Currency.php [hunk 1]
```diff
@@ -1126,7 +1126,7 @@ public static function countActiveCurrencies($idShop = null)
             self::$countActiveCurrencies[$idShop] = Db::getInstance()->getValue('
 				SELECT COUNT(DISTINCT c.id_currency) FROM `' . _DB_PREFIX_ . 'currency` c
 				LEFT JOIN ' . _DB_PREFIX_ . 'currency_shop cs ON (cs.id_currency = c.id_currency AND cs.id_shop = ' . (int) $idShop . ')
-				WHERE c.`active` = 1
+				WHERE c.`active` = 1 AND c.`deleted` = 0
```
Comment (PululuK) [inline classes/Currency.php:1129]: To be consistent with function name, I suggest to add a new optional bool params like `excludeDeleted` ?

**Review by PululuK — APPROVED** [review]

File: classes/Currency.php (hunk 1 above)
Comment (ghost) [reply classes/Currency.php:1129]: Why not, but this PR will not be testable anymore and I should make another one in the module?

What do you think about it?

Comment (Hlavtox) [reply classes/Currency.php:1129]: Function name is `countActiveCurrencies`, I think this parameter should not be needed.

**Review by kpodemski — APPROVED** [review]

**Review by florine2623 — APPROVED** [review]
Hello @okom3pom ,

Thanks for the PR.

Tested with existing currency and manually created currency.
Tested deletion and disabling of currency.
Display if OK when only 1 currency is enabled.

It is QA ✅

**Comment by jolelievre** [comment]
Thanks @okom3pom and @florine2623

**Pull request merged**